lib/dbservice/users.ex

```diff
@@ -224,6 +224,28 @@ defmodule Dbservice.Users do
     end
   end
 
+  @doc """
+  Updates a user first and then the student.
+
+  ## Examples
+
+      iex> update_student_with_user(%{field: value})
+      {:ok, %Student{}}
+
+      iex> update_student_with_user(%{field: bad_value})
+      {:error, %Ecto.Changeset{}}
+
+  """
+  def update_student_with_user(student, users, attrs \\ %{}) do
```

Comment:
is it supposed to be `users` or `user`? As I understand, a student is only connected to one user, right?